build.gradle

```diff
@@ -18,8 +18,8 @@ repositories {
 }
 
 dependencies {
-  compile "io.netty:netty-handler:4.1.0.Beta8"
-  compile "org.sharegov:mjson:1.3"
+  compile "io.netty:netty-handler:4.1.8.Final"
```

Comment:
Yes, gson is known to work well on Android. I don't know about the latest Netty-handler library. I don't have an Android device, can you test that?